assets/css/base/gutenberg-blocks.scss

```diff
@@ -330,10 +330,6 @@
 		margin: 0 0 ms(2);
 	}
 
-	div.wp-block-image {
-		display: inline;
-	}
-
```

Comment:
This was added to "fix floated image alignment", so will need to test to ensure floated images still work correctly.

Reply:
Shouldn't the CSS code for floated images be adjusted separately? It obviously broke the layout of the image block as seen on the screenshot above.

Reply:
Yes, we'll need to find a way to target these css rules appropriately. I tested all the align options for the image block with Storefront to see if there are any regressions. With this PR, the text flows (wraps) around `right`/`left` aligned image blocks in editor, but not in front end. 

<img width="1992" alt="Screen Shot 2020-01-31 at 10 54 30 AM" src="https://user-images.githubusercontent.com/4167300/73493915-8933d200-4418-11ea-9c33-198c702ffb7d.png">

In the current released version of Storefront, text does not wrap in front end or editor, so we'll need to resolve that. I'll take a closer look at this and see if we need to target the editor buttons specifically, and/or if we have a bug in left/right align that we need to log & fix.

Reply:
Tested the align options and this fix is working well, I couldn't find any regressions.